README.md

```diff
@@ -113,8 +113,8 @@ var _ = Service("calc", func() {
 ```
 
 This file contains the design for a `calc` service which accepts HTTP GET
-requests to `/add/{a}/{b}` where `{a}` and `{b}` are placeholders for integer
-values. The API returns the sum of `a` and `b` in the HTTP response body.
+requests to `/multiply/{a}/{b}` where `{a}` and `{b}` are placeholders for integer
+values. The API returns result of `a` multiplied by `b` in the HTTP response body.
```

Comment:
missing "the" (The API returns the result of...)

Reply:
Isn't it "the product"?

Reply:
Yes that would be even better "The API returns the product of ..."

Reply:
done 👍